### PR TITLE
FIx #32 Added new test system from the CMakeProject temaplte

### LINKED ISSUE #32: Backport new test system from the CMakeProject temaplte
I finished refactoring our tests system. 
Please add support of new test system for CopyrighFixer. see this [PR](https://github.com/QuasarApp/CMakeProject/pull/8) for get more information about changes.

### PR DESCRIPTION
**Pull request by IgorekLoschinin**

### CODE REVIEW

File: tests/tstMain.cpp [hunk 1]
```diff
@@ -8,25 +8,41 @@
...
+
+private:
+
+    /**
+     * @brief initTest This method prepare @a test for run in the QApplication loop.
+     * @param test are input test case class.
+     */
+    void initTest(Test* test);
+
+    QList<Test*> _tests;
```
Comment (EndrII) [inline tests/tstMain.cpp:44]: This field is not needed anymore
```suggestion
```

File: tests/tstMain.cpp [hunk 2]
```diff
@@ -37,46 +53,31 @@ private slots:
...
+    delete _app;
+}
 
-    app.exec();
+void tstMain::initTest(Test *test) {
+    QTimer::singleShot(0, this, [this, test]() {
+        test->test();
+        delete test;
+        _app->exit(0);
+});
```
Comment (EndrII) [inline tests/tstMain.cpp]: please correct indents

**Review by EndrII — CHANGES_REQUESTED** [review]

File: tests/tstMain.cpp (hunk 2 above)
Comment (IgorekLoschinin) [reply tests/tstMain.cpp]: Done.

**Review by EndrII — APPROVED** [review]
can be merged

**Pull request merged**